local/gugcat/add/index.php

```diff
@@ -41,7 +41,7 @@
 $activityid = required_param('activityid', PARAM_INT);
 $studentid = required_param('studentid', PARAM_INT);
 $course = $DB->get_record('course', ['id' => $courseid], '*', MUST_EXIST);
-$student = $DB->get_record('user', array('id'=>$studentid, 'deleted'=>0), '*', MUST_EXIST);
```

Comment:
@natividadelijah  I think the get_record is enough to get the info of the student in add form.